docs/loading/Spark-connector-starrocks.md

```diff
@@ -0,0 +1,278 @@
+# Load data using Spark connector (recommended)
+
+StarRocks provides a self-developed connector named StarRocks Connector for Apache Spark™ (Spark connector for short) to help you load data into a StarRocks table by using Spark. The basic principle is to accumulate the data and then load it all at a time into StarRocks through [STREAM LOAD](../sql-reference/sql-statements/data-manipulation/STREAM%20LOAD.md). The Spark connector is implemented based on Spark DataSource V2. A DataSource can be created by using Spark DataFrames or Spark SQL. And both batch and structured streaming modes are supported.
+
+## Version requirements
+
+| Spark connector | Spark            | StarRocks     | Java | Scala |
+| --------------- | ---------------- | ------------- | ---- | ----- |
+| 1.1.0           | 3.2, 3.3, or 3.4 | 2.5 and later | 8    | 2.12  |
+
+## Obtain Spark connector
+
+You can obtain the Spark connector JAR file in the following ways:
+
+- Directly download the compiled Spark Connector JAR file.
+- Add the Spark connector as a dependency in your Maven project and then download the JAR file.
+- Compile the source code of the Spark Connector into a JAR file by yourself.
+
+The naming format of the Spark connector JAR file is `starrocks-spark-connector-${spark_version}_${scala_version}-${connector_version}.jar`.
+
+For example, you install Spark 3.2 and Scala 2.12 in your environment, and you want to use Spark connector 1.1.0. You can use `starrocks-spark-connector-3.2_2.12-1.1.0.jar`.
```

Comment:
unnatual flow.
suggest changing the sentence to "For example, if you install Spark 3.2 and Scala 2.12 in your environment and you want to use Spark connector 1.1.0, you can use `starrocks-spark-connector-3.2_2.12-1.1.0.jar`." OR "For example, you install Spark 3.2 and Scala 2.12 in your environment, and you want to use Spark connector 1.1.0. In this situation, you can use `starrocks-spark-connector-3.2_2.12-1.1.0.jar`."